antora.yml

```diff
@@ -25,7 +25,7 @@ asciidoc:
     # for the project
     che-plugin-registry-directory: che-plugin-registry
     devworkspace-operator-index-disconnected-install: quay.io/devfile/devworkspace-operator-index:release-digest
-    devworkspace-operator-version-patch: "0.25.0"
+    devworkspace-operator-version-patch: ""
```

Comment:
empty version for dwo, something went wrong in release workflow

Reply:
it should be 0.27, correct?

Reply:
0.27 is not out yet, so this one should be 0.26.0

similarily, it is so in related chectl PR
https://github.com/che-incubator/chectl/pull/2867/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR3674

Reply:
@deerskindoll I updated the value manually in this PR to "0.26.0"

And for future runs, this should be fixed by this PR:
https://github.com/eclipse-che/che-docs/pull/2715